spec/services/unpublish_service_spec.rb

```diff
@@ -47,6 +47,22 @@
       expect(edition.status.details.public_explanation).to eq(public_explanation)
     end
 
+    it "saves the published_status to the withdrawal record" do
+      UnpublishService.new.withdraw(edition, public_explanation, user)
+      edition.reload
+
+      expect(edition.status.details.published_status.state).to eq("published")
+    end
+
+    it "saves the correct published_status when the document is already withdrawn" do
```

Comment:
Again, would it be better to compare the statuses here?

Reply:
See above comment ^

Reply:
See above comment below your comment ^˅ ;-).

Reply:
done the comment above below your comment ;)

Reply:
Should we actually be checking the `withdrawn_edition.status.details.published_status` here? I think what might help is if we clarify the 'it' for this spec, as 'correct' is a bit ambiguous :-|.